app/representers/api/distribution_representer.rb

```diff
@@ -5,6 +5,18 @@ class Api::DistributionRepresenter < Api::BaseRepresenter
   property :guid
   property :url
   property :kind
+
+  property :set_audio_version_template_uris,
+           readable: false,
+           reader: ->(doc, _args) do
+             distribution_templates.clear
```

Comment:
So you're saying the frontend shouldn't just PUT this every time then, right?  Only when it changes.  Since this is going to recreate the join table rows, whether or not it actually changed.

Reply:
Also wondering what the current set `audio_version_template` will do once the has-one relation disappears.

Reply:
(To be clear, i'm fine with a PUT always replacing them.  Just making sure I know the API!)

Reply:
Not sure I can make everything work correctly for both single and multiple relation here.
I was assuming that these change infrequently enough that this is a good strategy.
How after are you saving changes to the distribution?

Reply:
For some properties, they're PUT every time you save.  For others (and i _think_ the `set_something_uri` are in this category) they only get serialized when changed.  Will just have to make sure this array-of-uris ends up in that category.

So given the complexity of making both the has-one and has-many work concurrently, it sounds the order of merge/deploy should be:

1. This
2. PRX/publish.prx.org#456 (currently making sure that plays well with these changes)
3. PRX/prx.org#306

Right?